npm install fails

Please update your packages

```
npm ERR! peerinvalid The package react@15.1.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-measure@0.3.5 wants react@>0.13.0
npm ERR! peerinvalid Peer react-motion@0.4.4 wants react@>=0.13.2 || ^0.14 || ^15.0.0
npm ERR! peerinvalid Peer react-dom@0.14.8 wants react@^0.14.8
```

--- a/package.json
+++ b/package.json
@@ -47,7 +47,7 @@
     "lodash": "^3.10.0",
     "node-libs-browser": "^0.5.2",
     "node-sass": "^3.2.0",
-    "react-dom": "^0.15.0",
+    "react-dom": "^15.0.0",
     "sass-loader": "^1.0.2",
     "style-loader": "^0.12.3",
     "webpack": "^1.9.12",

--- a/package.json
+++ b/package.json
@@ -47,7 +47,7 @@
     "lodash": "^3.10.0",
     "node-libs-browser": "^0.5.2",
     "node-sass": "^3.2.0",
-    "react-dom": "^0.14.0",
+    "react-dom": "^0.15.0",
     "sass-loader": "^1.0.2",
     "style-loader": "^0.12.3",
     "webpack": "^1.9.12",

